src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Services/AzureSqlElasticPoolCommunicator.cs

```diff
@@ -103,9 +103,9 @@ public IList<Management.Sql.Models.ElasticPool> List(string resourceGroupName, s
         }
 
         /// <summary>
-        /// Creates or updates an Elastic Pool
+        /// Creates an Elastic Pool
         /// </summary>
-        public Management.Sql.Models.ElasticPool CreateOrUpdate(string resourceGroupName, string serverName, string elasticPoolName, Management.Sql.Models.ElasticPool parameters)
+        public Management.Sql.Models.ElasticPool Create(string resourceGroupName, string serverName, string elasticPoolName, Management.Sql.Models.ElasticPool parameters)
```

Comment:
This method does CreateOrUpdate, so it should be called CreateOrUpdate

Reply:
@jaredmoo but this method will only be used when create pool. when update pool, we call the other method using patch. So I was thinking of to distinguish two, call this Update?

Reply:
IMO it is more clear to just use the same name as the SDK method. But it is not that important. Your choice